packages/bot/src/handlers/members/GUILD_MEMBERS_CHUNK.ts

```diff
@@ -11,7 +11,7 @@ export async function handleGuildMembersChunk(bot: Bot, data: DiscordGatewayPayl
   // If this request has no nonce, skip checks.
   if (!payload.nonce) return
 
-  const pending = bot.gateway.cache.requestMembers.pending.get(payload.nonce)
+  const pending = bot.gateway.cache.requestMembers.pending!.get(payload.nonce)
```

Comment:
rather than suppressing the null errors, it would probably be best to update the GatewayManager interface to remove the nullable from cache.requestMembers.pending